apps/frontend/src/network/users/state.ts

```diff
@@ -35,17 +36,31 @@ const userState = selectorFamily<UserResponse | undefined, string>({
 
 export const useUserById = (id: string) => useRecoilValue(userState(id));
 export const usePatchUserById = (id: string) => {
+  const { getTokenSilently } = useAuth0();
   const authorization = useRecoilValue(authorizationState);
   const setPatchedUser = useSetRecoilState(patchedUserState(id));
   return async (patch: UserPatchRequest) => {
     setPatchedUser(await patchUser(id, patch, authorization));
+    await getTokenSilently({
```

Comment:
Sorry to start another round of this, but ... test? :D
E.g. do a `Provider` that re-provides the auth0 context with `getTokenSilently` overriden with a mock. Then a test case similar to the one checking the patch happening, except it checks that this is called